pkgs/top-level/all-packages.nix

```diff
@@ -370,6 +370,8 @@ with pkgs;
 
   bootstrap-studio = callPackage ../development/web/bootstrap-studio { };
 
+  borealis-cursors = callPackage ../data/icons/borealis-cursors { };
```

Comment:
Can you use the new by-name structure https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#name-based-package-directories ?